addon-test-support/poll-task-for.ts

```diff
@@ -1,12 +1,15 @@
 import { join } from '@ember/runloop';
 import { assert } from '@ember/debug';
-import { queuedPollTasks } from 'ember-lifeline';
+import { settled } from '@ember/test-helpers';
+import { queuedPollTasks, Token } from 'ember-lifeline';
 
-export default function pollTaskFor(token) {
+export default async function pollTaskFor(token: Token) {
```

Comment:
1. we don't use `await` so we dont' need `async` here
2. we can't rely on the host application to have regenerator runtime

```suggestion
export default function pollTaskFor(token: Token) {
```